### ISSUE TITLE
Exception for plugin <Spotify> when query <sp>

### ISSUE THREAD

**Issue by waaverecords** [waaverecords/PowerToys-Run-Spotify#4]
Same here.

I get the following error:
```
-------------------------- Begin exception --------------------------
Message: Exception for plugin <Spotify> when query <sp>

Exception full name  : System.ArgumentException
Exception message    : String is empty or null (Parameter 'query')
Exception stack trace:
   at SpotifyAPI.Web.Ensure.ArgumentNotNullOrEmptyString(String value, String name)
   at SpotifyAPI.Web.SearchRequest..ctor(Types type, String query)
   at PowerToys_Run_Spotify.Main.Query(Query query) in C:\Users\m_sno\PowerToys-Run-Spotify\Main.cs:line 96
   at PowerLauncher.Plugin.PluginManager.<>c__DisplayClass23_0.<QueryForPlugin>b__0()
   at Wox.Infrastructure.Stopwatch.Debug(String message, Action action)
   at PowerLauncher.Plugin.PluginManager.QueryForPlugin(PluginPair pair, Query query, Boolean delayedExecution)
Exception source     : SpotifyAPI.Web
Exception target site: Void ArgumentNotNullOrEmptyString(System.String, System.String)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------
```

_Originally posted by @matheustenedini in https://github.com/waaverecords/PowerToys-Run-Spotify/issues/2#issuecomment-1898360093_

**Comment by waaverecords** [comment]
This most likely happens when typing the activation keyword, and then typing nothing (or spaces) afterward.

But this probably doesn't crash the plugin and only writes an exception into logs.

**Comment by waaverecords** [comment]
Fixed in 0441e2d3f2601cd2257ed6192882e8bd3e492fd3.
Will be in the next release.
